package.json

```diff
@@ -38,7 +40,7 @@
     "ember-load-initializers": "^0.5.1",
     "ember-resolver": "^2.0.3",
     "loader.js": "^4.0.1",
-    "release-it": "~10.4.5"
+    "release-it": "^11.0.2"
```

Comment:
After adding these deps I believe we need you to run `yarn` and add the updated lockfile to this commit.

Reply:
Updated :)